www/index.html

```diff
@@ -1,56 +1,85 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Hello wasm-nu</title>
-    <link rel="stylesheet" href="https://fonts.xz.style/serve/inter.css" />
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css"
-    />
-
-    <style>
-      #examples {
-        display: flex;
-        column-gap: 1rem;
-      }
-      .output-error {
-        font-family: monospace;
-      }
-      .output-error--inline {
-        color: red;
-      }
-      td {
-        min-width: 6rem;
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Hello wasm-nu</title>
+  <link rel="stylesheet" href="https://fonts.xz.style/serve/inter.css" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css" />
+
+  <style>
+    #examples {
+      display: flex;
+      column-gap: 1rem;
+    }
+
+    .output-error {
+      font-family: monospace;
+    }
+
+    .output-error--inline {
+      color: red;
+    }
+
+    td {
+      min-width: 6rem;
+    }
+  </style>
+</head>
+
+<body>
+  <header>
+    <h1>wasm-nu</h1>
+  </header>
+  <noscript>This page contains webassembly and javascript content, please enable
+    javascript in your browser.</noscript>
+
+  <script type="text/javascript" src="browserfs.min.js"></script>
+  <script type="text/javascript">
+    // Installs globals onto window:
+    // * Buffer
+    // * require (monkey-patches if already defined)
+    // * process
+    // You can pass in an arbitrary object if you do not wish to pollute
+    // the global namespace.
+    BrowserFS.install(window);
```

Comment:
Is there a reason for this being inline in the html file? If not, can we move it to bootstrap.js? In there we could then move the `import()` call into the `BrowserFS.configue` callback.

Reply:
=> https://github.com/nushell/wasm-nu/pull/24